docs/architecture-decisions/adr006-avoid-react-fc.md

```diff
@@ -0,0 +1,42 @@
+# ADR006: Avoid React.FC and React.SFC
+
+## Context
+
+Facebook has removed ```React.FC``` from their base template for a Typescript project. The reason for this was that it was found to be an unnecessary feature with next to no benefits in combination with a few downsides.
+
+The main reasons were:
+- **children props** were implicitly added
+- **Generic Type** lacked support
+
+Read more about the removal in [this PR](https://github.com/facebook/create-react-app/pull/8177).
+
+## Decision
+
+To keep our codebase up to date, we have decided that React.FC and React.SFC should be avoided in our codebase when adding new code.
+
+Here is an example:
+````
```

Comment:
hm, should just be three backticks, but it seems like it works anyway? hm

but even better, do ` ```ts ` so that you get syntax highlighting

Reply:
Thanks for the tip! Will do.